### ISSUE TITLE
keyword search for quantifiers 

### ISSUE THREAD

**Issue by divine-comedian** [commons-stack/praise#422]
I LOVE this new header for marking duplicates! 

gathering feedback from quantifiers it would be awesome to implement a keyword search to make marking duplicates and scoring easier. 

having a text field right in the duplicate/dismiss header that users could input a keyword and it would highlight all matching text on the page. marking duplicates or dismissing or doing any interactions on the page would not reset or remove the keywords input.

![image](https://user-images.githubusercontent.com/67759413/170800406-497e56db-5334-4834-b6b3-8effd363a53e.png)

**Comment by kristoferlund** [comment]
Yes, that is definitely the plan. This was the original wireframe: 

<img width="627" alt="image" src="https://user-images.githubusercontent.com/9698363/171013696-a8af7cea-d76f-4654-8dbd-f0b018e18231.png">

**Comment by kristoferlund** [comment]
- Filter list based on keyword
- Select all
- Set quantify score on more than one item at the same time
